test/api/api-test.js

```diff
@@ -654,6 +654,10 @@ describe('JsonApi', () => {
 
       jsonApi.destroy('foo', 1).then(() => done()).catch(() => done())
     })
+
+    it.skip('should throw an error while attempting to access undefined model', (done) => {
```

Comment:
What's the thinking here? Not sure about including a test is skipped

Reply:
Because at the time I couldn't figure out how to write a unit test which expects an error but I wanted to include a representative test in the PR, I think I have since figured it out so I'll give it a shot.